tests/Functional/App/config.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+
+<container xmlns="http://symfony.com/schema/dic/services"
```

Comment:
I get the XML sympathy in this project, doesn't look exactly friendly to replicate into a custom application?

Reply:
The application can use XML, YAML or PHP. I'm just using XML here since it's a little different (eg should have a schema).